refinery/ui/source/js/user-file-browser/services/user-file.js

```diff
@@ -40,8 +40,14 @@
             },
             sort: function () {
               var sort = userFileSortsService.fields.map(function (field) {
-                return field.name + '_Characteristics_generic_s ' + field.direction
-                    + ', ' + field.name + '_Factor_Value_s ' + field.direction;
+                var name = field.name;
+                var dir = field.direction;
+                if (name === 'filename') {
```

Comment:
What is `dir`? I'd avoid abbreviations for clarity or simply not use those extra intermediate variables.